### ISSUE TITLE
pods stuck in terminated stated when using istio-injector

### ISSUE THREAD

**Issue by prune998** [istio/istio#5067]
Hi there. I'm having this issue since upgrade to 1.9.4 and 1.9.6.
I opened a case to GKE support team, which finally openend a public issue.

It's not on the Istio side but I'm referencing it here so people may find it in case they reach the issue.

https://github.com/kubernetes/kubernetes/issues/62666

Solution in the meantime is either : 
- do not use the auto-injector for Istio sidecar
- remove the LimitRange

Ex of pod list during issue : 

```
kubectl get pods
NAME                               READY     STATUS        RESTARTS   AGE
greeter-server-5f4b8df8df-sb9tn    0/2       Terminating   0          7d
greeter-server-5f4b8df8df-vdkxf    0/2       Terminating   0          6d
greeter-server-658bbbbb9-829hx     0/2       Terminating   0          2d
greeter-server-658bbbbb9-qsm5x     0/2       Terminating   0          4d
greeter-server-658bbbbb9-zdnwz     2/2       Running       0          2d
loadtest-client-549d98cc6c-tp54d   1/1       Running       0          4d
loadtest-client-778cc6b5d7-9722x   0/2       Terminating   0          4d
loadtest-client-778cc6b5d7-tmggh   0/2       Terminating   0          4d
```

You can correct this by : 
```
kubectl delete limitrange limits
```

It can take up to 15 mins for the Garbage collector to remove all the pods. 
I tried putting the limit back after cleanup and it was working fine then, but I did not took time to ensure that.
K8s will come with a solution, hopefully in the next release.

**Comment by ostromart** [comment]
We need to find the correct place to capture this. Currently we only have official docs which is not appropriate. Maybe istio-users to make it searchable. Moving to P1.

**Comment by prune998** [comment]
I think this is solved with Istio 0.8.0 and K8s 1.10.2.
I think it was linked to a k8s 1.9.x issue. And Isito 0.8.0 does not seem to create the `limitrange`...

**Comment by ostromart** [comment]
Thanks, closing this.
